bal_tools/models.py

```diff
@@ -1,9 +1,8 @@
 from typing import Optional, List, Tuple, Dict, NewType
 from decimal import Decimal
-from dataclasses import dataclass, fields
+from dataclasses import dataclass
 from enum import Enum
 from pydantic import BaseModel, field_validator, model_validator, Field
-import json_fix
```

Comment:
json_fix will need to imported in order for the `CorePools.__json__` hook work. i think the import is misleading since it looks like the module isnt doing anything. maybe can add a comment to this for clarification

Reply:
dd230ef